client/state/plugins/installed/status/reducer.js

```diff
@@ -27,6 +27,13 @@ import {
 	PLUGIN_REMOVE_REQUEST_FAILURE,
 } from 'calypso/state/action-types';
 
+/**
+ * Plugin installation statuses
+ */
+export const PLUGIN_INSTALLATION_IN_PROGRESS = 'inProgress';
```

Comment:
These would be really useful outside of the reducer, as they're used in multiple areas in plugins. We have a constants file in `lib/plugins` that seem like a good place for these. Another one could be if we create a constants file in here.

In general, it feels really odd for `actions` to import these from the `reducer`. In any case, let's define these in a separate file that we can import both here and in the actions file.

Reply:
I will create a constants file here! 👍